### PR TITLE
feat(drive-abci)!: processing fees for signature verification

### PR DESCRIPTION
**Pull request by QuantumExplorer**
## Issue being fixed or feature implemented
Feature implements fee charging for signature verification in state transitions.

## What was done?
Added validation operations for every state transitions:

## How Has This Been Tested?
Fees increased on a few tests.

## Breaking Changes
This will break with testnet, as fees for operations are changing.

## Checklist:

- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have added or updated relevant unit/integration/functional/e2e tests
- [X] I have added "!" to the title and described breaking changes in the corresponding section if my code contains any
- [X] I have made corresponding changes to the documentation if needed

**For repository code-owners and collaborators only**
- [ ] I have assigned this pull request to a milestone

### CODE REVIEW

File: packages/rs-drive-abci/src/execution/types/execution_operation/mod.rs [hunk 1]
```diff
@@ -8,12 +12,54 @@ pub mod signature_verification_operation;
 
 #[derive(Debug, Clone, PartialEq, Eq)]
 pub enum ExecutionOperation {
+    DoubleSha256,
```
Comment (markin-io) [inline packages/rs-drive-abci/src/execution/types/execution_operation/mod.rs:15]: Don't we need Single SHA256 as well?

File: packages/rs-drive-abci/src/execution/platform_events/state_transition_processing/validate_fees_of_event/v0/mod.rs [hunk 2]
```diff
@@ -60,6 +64,13 @@ where
                     )
                     .map_err(Error::Drive)?;
 
+                ExecutionOperation::add_many_to_fee_result(
```
Comment (shumkov) [inline packages/rs-drive-abci/src/execution/platform_events/state_transition_processing/validate_fees_of_event/v0/mod.rs:67]: Did you consider to add this method to FeeResult?

File: packages/rs-drive-abci/src/execution/types/execution_operation/mod.rs (hunk 1 above)
Comment (QuantumExplorer) [reply packages/rs-drive-abci/src/execution/types/execution_operation/mod.rs:15]: Well we don't use it anywhere right now, so no.

**Review by shumkov — APPROVED** [review]

File: packages/rs-drive-abci/src/execution/platform_events/state_transition_processing/validate_fees_of_event/v0/mod.rs (hunk 2 above)
Comment (QuantumExplorer) [reply packages/rs-drive-abci/src/execution/platform_events/state_transition_processing/validate_fees_of_event/v0/mod.rs:67]: We can not, as FeeResult is in DPP

**Review by markin-io — APPROVED** [review]

**Pull request merged**